gammapy/image/maps.py

```diff
@@ -252,6 +253,14 @@ def solid_angle(self):
         omega = -np.diff(xsky, axis=1)[1:, :] * np.diff(ysky, axis=0)[:, 1:]
         return Quantity(omega, 'deg2').to('sr')
 
+    def center(self):
+        """
+        Center coordinates of the sky map.
```

Comment:
I'm 50:50 on this.

- Property is a bit nicer / used much in other projects.
- Method is more extensible in a backward-compatible way (add keyword args)

@adonath - you make the decision on this and we can do a systematic review of the Gammapy API before 1.0.
:-)